Update Experiment PHP docs with AmplitudeCookie util (#93)
amplitude/amplitude-dev-center: docs/experiment/sdks/php-sdk.md

        // Flag is off
    }
}
```

## Access Amplitude cookies

> If you're using the Amplitude Analytics SDK on the client-side, the PHP server SDK provides an `AmplitudeCookie` class with convenience functions for parsing and interacting with the Amplitude identity cookie. This is useful for ensuring that the Device ID on the server matches the Device ID set on the client, especially if the client hasn't yet generated a Device ID.

Reviewer: ```suggestion
If you use the Amplitude Analytics SDK on the client-side, the PHP server SDK provides an `AmplitudeCookie` class with convenience functions for parsing and interacting with the Amplitude identity cookie. This helps ensure that the Device ID on the server matches the Device ID set on the client, especially if the client hasn't yet generated a Device ID.
```
